src/main/scala/edu/berkeley/ce/rockslicing/CommandReader.scala

```diff
@@ -0,0 +1,13 @@
+package edu.berkeley.ce.rockslicing
+
+/** A simple tool for parsing command line inputs
+  * @constructor Create a new command reader
+  */
+case class CommandReader(inputs: Array[String]) {
+  assert(inputs.length >= 2)
```

Comment:
I have it so that it prints out the usage and structure of arguments. Didn't go into detail about how to start a Spark instance, but just listed the website for Spark where they can get more info. Does this seem reasonable?